CHANGELOG.md

```diff
@@ -8,6 +8,8 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 ### Added
 
+- Users can edit the Academy URN on a project
```

Comment:
As these are primarily aimed at end users could we make this a bit more specific? Something like:

> User can provide the new academy urn once it has be created in Get information about schools